astropy_helpers/tests/test_setup_helpers.py

```diff
@@ -543,3 +543,196 @@ def compiler_setter_with_sysconfig(compiler):
 
         assert len(log.messages) == 1
         assert 'The C compiler used to compile Python' in log.messages[0]
+
+
+def test_adjust_compiler(monkeypatch, tmpdir):
```

Comment:
Why is this test repeated?

Reply:
Good catch, I think it's a copy/paste error.